marshal/claim.go

```diff
@@ -393,7 +397,7 @@ func (c *claim) healthCheck() bool {
 	// If were behind by too many cycles, then we should try to release the
 	// partition. If so, do this in a goroutine since it will involve calling out
 	// to Kafka and releasing the partition.
-	if c.cyclesBehind >= 3 {
+	if c.cyclesBehind >= 5 {
```

Comment:
Why change this to 5? I don't think we want a partition to be behind for 5 minutes before releasing it?

If you're going to change this, it should be a tuneable in the options structure.